### PR TITLE
feat(chat/native): fixed keyboard overlapping chat input bar

### LINKED ISSUE #13748: RNSDK - Android and iOS - Chat screen - incorrect margin/space between input text field and keyboard
### Description:

On mobile devices, using Jitsi RNSDK, chat input text box looks too far from keyboard on Android and overlapping on iOS.

Is it expected?

### Steps to reproduce:

1. Open mobile app developed using RNSDK (and React Native 0.72.4 - maybe other versions too)
2. Open chat
3. Start typing text

### PR DESCRIPTION
**Pull request by Calinteodor**
Fixes #13748.

1. Iphone 14 
Simulator Screenshot - iPhone 14 - 2023-10-25 at 13 36 31 
2. Iphone 14 PRO Max 
Simulator Screenshot - iPhone 14 Pro Max - 2023-10-25 at 13 36 20
3. Iphone SE 
Simulator Screenshot - iPhone SE (3rd generation) - 2023-10-25 at 13 36 16

### CODE REVIEW

File: react/features/chat/components/native/ChatInputBar.tsx [hunk 1]
```diff
@@ -66,11 +74,18 @@ class ChatInputBar extends Component<IProps, IState> {
...
+        let inputBarStyles;
+
+        if (this.props.aspectRatio === ASPECT_RATIO_WIDE) {
+            inputBarStyles = styles.inputBarWide;
+        } else {
+            inputBarStyles = styles.inputBarNarrow;
+        }
+
         return (
-            <SafeAreaView
```
Comment (saghul) [inline react/features/chat/components/native/ChatInputBar.tsx:70]: Why are we dropping this? Does it no longer help?

Comment (Calinteodor) [reply react/features/chat/components/native/ChatInputBar.tsx:70]: No, there is no need for it anymore.

Comment (saghul) [reply react/features/chat/components/native/ChatInputBar.tsx:70]: But we seem to position things using hardcoded offsets now right? Isn't that worse.

Comment (Calinteodor) [reply react/features/chat/components/native/ChatInputBar.tsx:70]: With or without safeareaview, we need to set a fixed height for this component. I can add back this safe area view and remove some of the height value.

**Review by saghul — APPROVED** [review]

File: react/features/chat/components/native/ChatInputBar.tsx (hunk 1 above)
Comment (saghul) [reply react/features/chat/components/native/ChatInputBar.tsx:70]: Ah nevermind, I see it was the wrapper of ChatInout now. Approved!

**Pull request merged**